backend/src/controllers/game.controller.ts

```diff
@@ -0,0 +1,44 @@
+import { NextFunction, Request, Response } from "express";
+import {
+  createGame as createGameService,
+  validateGameCode as validateGameCodeService,
+} from "../services/game.service";
+
+export const createGame = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<void> => {
+  try {
+    const dbResp = await createGameService();
+    console.log("Created game!");
+    res.status(201).send(dbResp);
+    return;
+  } catch (e) {
+    return next(e);
+  }
+};
+
+export const validateGame = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<void> => {
+  try {
+    const code = req.query.gameCode;
+    if (typeof code != "string") {
+      res.status(400).send("Code not of type string");
+      return;
+    }
+    const dbResp = await validateGameCodeService(code);
+    if (dbResp) {
```

Comment:
I think something like `isValid` or `gameExists: boolean` would be more descriptive than `dbResp`